script/Arcueid/Day 11/11_00_ARC11_1.txt

```diff
@@ -445,7 +445,7 @@ A wail goes up from the young men that were in the Noel faction.
 [sha:2425d37bc192d839ac35ddcb6958aa31eeb51dc0]{
 -- Page 27, Offset 29557.
 -- 俺は漠然と、そうか、と頷いた。
-I just nod absentmindedly.
+I just nod absent-mindedly.
```

Comment:
Revert, seems to be more common.

Reply:
ig I'll change `Whenever anyone spoke to me, I could only reply **absent-mindedly**.` then